.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             "git+https://github.com/cta-observatory/ctapipe_io_lst@$CTAPIPE_IO_LST_VERSION" \
             pytest-cov
 
-          pip install -e .
```

Comment:
I noticed that Travis and GitHub actions were running tests in one of the pull requests and found this one still not merged.

Just for curiosity, why is the `-e` option preventing from passing the tests?

Reply:
I don't think it is.

Reply:
But -e is also not needed for travis, no one will edit code there :wink: